### DISCUSSION TITLE
Javascript file not working on github pages

### DISCUSSION THREAD

**Discussion by Scorp1Oz** [community/community#30664] [Other Feature Feedback, Questions, & Ideas]
Hello Github,
https://github.com/Scorp1Oz/Print-Point
Javascript file not working on github pages.
how it works
![1](https://user-images.githubusercontent.com/111981572/186428020-38fda01d-1cae-4c46-b997-78bfe6e2a9ad.png)
how it should work
![2](https://user-images.githubusercontent.com/111981572/186428073-eb161d6d-ea12-480d-a019-c03633d8cd92.png)

**Comment by luigieterovik** [comment]
Are you sure you staged your JavaScript file before commiting? git add .

**Comment by GhostOf0days** [comment]
Could you please show the relevant source code or make it pseudocode?

**Comment by jshtuazon2** [comment]
51984344

**Comment by Scorp1Oz** [comment]
bruh thanks to all who try to help but it's was almost year ago.
